_layouts/mainhomepage.html

```diff
@@ -45,7 +45,7 @@ <h1>Jamulus</h1>
           <a href="wiki/Getting-Started" id="get_started_btn_top">{{ page.mTGetStartedNow }}</a>
         </div>
         <div class="fx-col-100-xs" id="quick_dl_container">
-          {{ page.mTDownloadNow }} <a href="{{ site.download_root_link }}{{ site.download_file_names.windows }}" target="_blank" rel="noreferrer" class="os-win">Windows</a>, <a href="{{ site.download_root_link }}{{ site.download_file_names.mac }}" target="_blank" rel="noreferrer" class="os-mac">Mac</a>, <a href="{{ site.download_root_link }}{{ site.download_file_names.deb-gui }}" target="_blank" rel="noreferrer" class="os-deb">Debian/Ubuntu</a> &amp; <a href="{{ site.download_overview_link }}" target="_blank" rel="noreferrer" class="os-other">{{ page.mTOtherPlatforms }}</a>.
+          {{ page.mTDownloadNow }} <a href="{{ site.download_root_link }}{{ site.download_file_names.windows }}" target="_blank" rel="noreferrer" class="os-win">Windows</a>, <a href="{{ site.download_root_link }}{{ site.download_file_names.mac }}" target="_blank" rel="noreferrer" class="os-mac">macOS</a>, <a href="{{ site.download_root_link }}{{ site.download_file_names.deb-gui }}" target="_blank" rel="noreferrer" class="os-deb">Debian/Ubuntu</a> and <a href="{{ site.download_overview_link }}" target="_blank" rel="noreferrer" class="os-other">{{ page.mTOtherPlatforms }}</a>.
```

Comment:
Thanks for that. Somehow this broke the site?

Reply:
Probably not a registered class somewhere?

Reply:
I think it's something bigger. As soon as we want to edit anything on the main page, po4a complains.

Reply:
There's an `a` missing in the tag after "write code" (it should be `</a>` instead of `</>`), line 46 in `1-Index.html`.